process/block/preprocess/transactions.go

```diff
@@ -1334,6 +1349,60 @@ func SortTransactionsBySenderAndNonce(transactions []*txcache.WrappedTransaction
 	sort.Slice(transactions, sorter)
 }
 
+// preFilterTransactions filters the transactions prioritising the move balance operations
```

Comment:
Do txs.preFilterTransactions. and you do not need to push gasHandler

Reply:
it's harder to test as I need to initialize a txspreprocessor

Reply:
You can easily do it with mocks and stubs.